tests/FakeItEasy.Analyzer.Tests.Shared/Helpers/ArgumentConstraintTestCases.cs

```diff
@@ -0,0 +1,18 @@
+namespace FakeItEasy.Analyzer.Tests.Helpers
+{
+    using Xunit;
+
+    public static class ArgumentConstraintTestCases
+    {
+        public static TheoryData<string> From(params string[] constraints)

```

Comment:
I didn't know about `TheoryData`! That's much better than `IEnumerable<object[]>`.

Reply:
Me niether! It's pretty exciting. As these things are rated.